.env.example

```
@@ -6,20 +6,23 @@
 # Set via `wrangler secret put <KEY>` for deployed environments.
 # For local dev, create packages/api/.dev.vars with these values.
 
-TURSO_DATABASE_URL=libsql://your-db.turso.io
-TURSO_AUTH_TOKEN=your_token_here
-JWT_SECRET=your_secret_here    # openssl rand -base64 32
-RESEND_API_KEY=re_your_key_here  # Resend email service
+TURSO_DATABASE_URL=
+TURSO_AUTH_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TURSO_AUTH_TOKEN key should go before the TURSO_DATABASE_URL key

(UnorderedKey)